docs/supported-libraries.md

```diff
@@ -30,6 +30,7 @@ These are the supported libraries and frameworks:
 | [Apache Dubbo](https://github.com/apache/dubbo/)                                                                                            | 2.7+                          | [opentelemetry-apache-dubbo-2.7](../instrumentation/apache-dubbo-2.7/library-autoconfigure)                                                                                                                                                                                                                                                                                             | [RPC Client Spans], [RPC Server Spans]                                                                                                  |
 | [Apache HttpAsyncClient](https://hc.apache.org/index.html)                                                                                  | 4.1+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [HTTP Client Spans], [HTTP Client Metrics]                                                                                              |
 | [Apache HttpClient](https://hc.apache.org/index.html)                                                                                       | 2.0+                          | [opentelemetry-apache-httpclient-4.3](../instrumentation/apache-httpclient/apache-httpclient-4.3/library)                                                                                                                                                                                                                                                                               | [HTTP Client Spans], [HTTP Client Metrics]                                                                                              |
+| [Apache HttpClient 5](https://hc.apache.org/index.html)                                                                                     | 2.0+                          | [opentelemetry-apache-httpclient-5.2](../instrumentation/apache-httpclient/apache-httpclient-5.2/library)                                                                                                                                                                                                                                                                               | [HTTP Client Spans], [HTTP Client Metrics]                                                                                              |
```

Comment:
I think you don't need to list `Apache HttpClient 5` separately just add the link to the library to the `Apache HttpClient` entry.  For example `Java Platform` also has multiple libraries.